src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Consider renaming folder/namspace from "BindingExtensions" to "Extensibility". We actually already have folder Binding\Extensibility - maybe these types should just go there?

Reply:
✔️